wirecard/oxid-ee
# WDPD-18 Check payment method config

### This PR

* Adds the functionality to check if a payment method’s credentials are valid

### How to Test

* In the menu under “Wirecard Configuration” select “Payment methods” and then choose the “Wirecard Paypal” method. Under the “HTTP Password” field there is a “Test credentials” button. After clicking it, a text should appear next to it whether the credentials are valid or not.

### Jira Links

* https://jira.parkside.at/browse/WDPD-18
* https://jira.parkside.at/browse/WDPD-93
